starport/pkg/spn/spn.go

```diff
@@ -142,65 +127,65 @@ func (c *Client) makeSureAccountHasTokens(ctx context.Context, address string) e
 }
 
 // handleBroadcastResult handles the result of broadcast messages result and checks if an error occurred
-func (c *Client) handleBroadcastResult() error {
-	out := struct {
-		Code int    `json:"code"`
-		Log  string `json:"raw_log"`
-	}{}
-	if err := json.NewDecoder(c.out).Decode(&out); err != nil {
+func handleBroadcastResult(resp *types.TxResponse, err error) error {
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			return errors.New("make sure that your SPN account has enough balance")
```

Comment:
May not be related to this PR but I believe that we check "not found" because an address that didn't receive tokens doesn't exist.

Therefore, I would rather return something like `make sure to provide tokens to your address` because the address has simply no balance

Reply:
this is a technical detail in the sdk. an account that does not exist on the chain means it has zero balances. user does not need to know the difference between if they emptied their account to zero balance or never had any balances.